lib/src/util/vulnerabilitySeverityRatings.ts

```diff
@@ -62,11 +62,14 @@ export const formatVulnerabilityScore = (scoreNumber: number): VulnerabilityScor
 };
 
 /**
- * Gets the CVSS severity rating category for a given CVSS score string.
- * Returns null if the score is not within the valid range.
+ * Gets the corresponding CVSS severity rating for a given CVSS score string or number.
+ * Throws an error if the score is not within the valid range or is not a valid number with one decimal place.
  */
-export const getVulnerabilitySeverityRating = (cvssScore: VulnerabilityScore): VulnerabilitySeverityRating | null => {
-  const score = parseFloat(cvssScore);
+export const getVulnerabilitySeverityRating = (cvssScore: VulnerabilityScore | number): VulnerabilitySeverityRating => {
+  const score = parseFloat((typeof cvssScore === 'number') ? formatVulnerabilityScore(cvssScore) : cvssScore);
```

Comment:
Is this passing `cvssScore` through `parseFloat` even when it's already a `number`? That would seem unnecessary

Reply:
> Is this passing cvssScore through parseFloat even when it's already a number? That would seem unnecessary

Oh wait I see, you're converting it from a number to a string and then back to a number.  Why? Surely there's a more efficient way to get whatever behavior you're trying to get from that operation.

Reply:
> > Is this passing cvssScore through parseFloat even when it's already a number? That would seem unnecessary
> 
> Oh wait I see, you're converting it from a number to a string and then back to a number. Why? Surely there's a more efficient way to get whatever behavior you're trying to get from that operation.

I've improved the impl here: [9da14ac](https://github.com/sonatype/sonatype-react-shared-components/pull/1202/commits/9da14acabcee920848ac7dcdfa17e2a582c7a715)